content/source/docs/cloud/guides/recommended-practices/part3.2.html.md

```diff
@@ -32,7 +32,7 @@ Make sure you've picked a VCS system that Terraform Cloud will be able to access
 
 Start moving infrastructure code into version control. New Terraform code should all be going into version control; if you have existing Terraform code that’s outside version control, start moving it in so that everyone in your organization knows where to look for things and can track the history and purpose of changes.
 
--> **Note:** There are several ways to structure Terraform repositories. If you want to learn more about how your repo structure can affect your Terraform Cloud workflows, see [VCS Repository Structure](/docs/cloud/workspaces/repo-structure.html) in the Terraform Cloud documentation.
+Some organizations prefer to keep each Terraform configuration in its own VCS repository, while others prefer to keep all configurations in a shared "monorepo." Both approaches are valid; if your organization doesn't already have a strong preference, we recommend separate repositories.
```

Comment:
do you think it would make sense to include more of the information from the "manageable repos" section of the repo-structure page here? 
that section seems like valuable information and i'm not sure if is easy to find anywhere else.

Reply:
I think the "manageable repos" stuff is still relevant, but since it's more of a philosophical/best-practices thing, it's kind of a poor fit for the main docs. But, Rachel's been writing a series of guides about code organization that should cover all of that ground in much more detail; once those are live, I think it'd make sense to link them from here.